status-im/status-react
# [#4663] Added collectibles and CK support

fixes #4663
fixes #4666

### Summary:

Added wallet support for collectibles and CryptoKitties details view.

### Steps to test:
- Open Status
- Make sure you have kitties
- Browse wallet
- Enable CK asset

Note: kitty images are not supported yet due to #4709

status: ready
